execution/executor-benchmark/src/lib.rs

```diff
@@ -155,21 +164,27 @@ pub fn run_benchmark<V>(
 
 fn init_workload<V, P: AsRef<Path>>(
     transaction_type: TransactionType,
+    num_main_signer_accounts: usize,
     db: DbReaderWriter,
     db_dir: &P,
+    pipeline_config: PipelineConfig,
 ) -> Box<dyn TransactionGeneratorCreator>
 where
     V: TransactionBlockExecutor<BenchmarkTransaction> + 'static,
 {
     let version = db.reader.get_latest_version().unwrap();
     let (pipeline, block_sender) =
-        Pipeline::<V>::new(BlockExecutor::new(db.clone()), version, false);
+        Pipeline::<V>::new(BlockExecutor::new(db.clone()), version, pipeline_config);
 
     let runtime = Runtime::new().unwrap();
 
     let num_existing_accounts = TransactionGenerator::read_meta(db_dir);
-    let num_cached_accounts = std::cmp::min(num_existing_accounts, MAX_ACCOUNTS_INVOLVED_IN_P2P);
-    let mut accounts_cache = TransactionGenerator::gen_user_account_cache(num_cached_accounts);
+    // let num_cached_accounts = std::cmp::min(num_existing_accounts, num_main_signer_accounts);
```

Comment:
nit: please remove

Reply:
Already modifying in the following PR, so will leave it as if for a bit